### DISCUSSION TITLE
keycloak-server-overlay. Where is it?

### DISCUSSION THREAD

**Discussion by flashboss** [keycloak/keycloak#12876] [Archive]
Hi, until the 15.1.1 keycloak version I used to import in my maven project the keycloak-server-overlay.zip (https://mvnrepository.com/artifact/org.keycloak/keycloak-server-overlay) . After the 15.1.1 it is not more. Where is it?

**Comment by flashboss** [comment]
are there news?

**Reply by bs-matil** [reply]
The code is still there but it had not been released since than https://github.com/keycloak/keycloak/blob/bfce612641a70e106b20b136431f0e4046b5c37f/distribution/server-overlay/pom.xml

**Reply by flashboss** [reply]
misses 17 too. It is based still on wildfly. Someone has sleeping

**Reply by bs-matil** [reply]
Could be intentional. But I don't know.
